build.gradle

```diff
@@ -133,6 +133,10 @@ publishing {
     }
 }
 
+test {
+    useJUnitPlatform()
```

Comment:
Afaik the `Junit` *should* be the default, so this shouldn't be necessary.
But it cannot hurt to declare this explicitly. 👍🏻